css/src/components/table.css

```diff
@@ -0,0 +1,168 @@
+.yoast table {
+	font-size: var(--font-size-default);
+	color: var(--color-default);
+	line-height: 1.2;
+	width: 100%;
+	border-bottom: var(--border-default);
+	border-spacing: 0;
+}
+
+.yoast tfoot th {
+	border-bottom: none;
+}
+
+.yoast th {
+	border-bottom: var(--border-default);
+}
+
+.yoast th, .yoast th.sortable {
+	color: var(--color-dark);
+	text-align: left;
+	padding: 16px;
+}
+
+.yoast th.sortable a, .yoast th a {
+	color: var(--color-dark);
+	padding: 0;
+	font-weight: 500;
+}
+
+.yoast tr {
+	height: 50px;
+}
+
+.yoast td {
+	padding: 16px;
+	border-bottom: var(--border-default);
+	font-size: var(--font-size-default);
+}
+
+.yoast .tablenav {
+	height: auto;
+	margin: 6px 0 16px;
+}
+
+.yoast .tablenav.top .tablenav-pages {
+	display: none;
+}
+
+.yoast .tablenav.top #posts-filter .tablenav-pages {
+	display: block;
+}
+
+.yoast .tablenav .yoast-button {
+	color: var(--color-dark);
+}
+
+.yoast .tablenav .search-box{
+	float: right;
+}
+
+.yoast .tablenav .search-box div {
+	float: left;
+	width: 200px;
+	margin-right: 8px;
+}
+
+.yoast .tablenav .displaying-num {
+	margin-right: 22px;
+}
+
+.yoast .tablenav .paging-input {
+	margin-right: 6px;
+}
+
+.yoast .column-actions {
+	text-align: right;
+	width: 200px;
+}
+
+.yoast .tablenav .tablenav-pages .button,
+.yoast .tablenav .tablenav-pages .button.disabled,
+.yoast .tablenav .tablenav-pages .tablenav-pages-navspan {
+	display: inline-block;
+	position: relative;
+	text-decoration: none;
+	border-radius: 4px;
+	transition: background-color 150ms ease-out 0s;
+	cursor: pointer;
```

Comment:
This will make the `<span>` element used for "disabled" navigation look like it's clickable, which isn't ideal:

<img width="410" alt="Screenshot 2020-06-11 at 13 14 15" src="https://user-images.githubusercontent.com/1682452/84379137-9c845480-abe5-11ea-90f2-54fb16403267.png">